config/purposes/ultima_pcr_free.wip.yml

```diff
@@ -36,3 +36,29 @@ UPF EqVol Norm:
   :creator_class: LabwareCreators::TubeFromTube
   :presenter_class: Presenters::FinalTubePresenter
   :default_printer_type: :tube
+UPF Balanced Pool:
+  :asset_type: tube
+  :target: StockMultiplexedLibraryTube
+  # TODO: IlluminaHtp type doesn't seem right for Ultima sequencing
+  :type: IlluminaHtp::InitialStockTubePurpose
```

Comment:
It provides a method sibling_tubes after pooling in case multiple pool tubes are generated, which is exposed on the tube as sibling_tubes through the API.